foxpuppet/tests/test_browser_model.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from foxpuppet import FoxPuppet
+
+
+# Tests opening a new private browsing window via menu
```

Comment:
Yeah, if tests need this extra level of explanation, let's make it a [docstring](https://www.python.org/dev/peps/pep-0257/) such as:

```python
def test_new_private_window(selenium):
    """Tests opening a new private browsing window via menu"""
```